tests/integration/deploy/test_deploy_command.py

```diff
@@ -929,6 +929,245 @@ def test_deploy_nested_stacks(self, template_file):
         # verify child stack ChildStackX's creation
         self.assertRegex(process_stdout, r"CREATE_COMPLETE.+ChildStackX")
 
+    @parameterized.expand(["aws-dynamodb-error.yaml"])
+    def test_deploy_create_failed_rollback(self, template_file):
+        template_path = self.test_data_path.joinpath(template_file)
+
+        stack_name = self._method_to_stack_name(self.id())
+        self.stacks.append({"name": stack_name})
+
+        deploy_command_list = self.get_deploy_command_list(
+            template_file=template_path,
+            stack_name=stack_name,
+            capabilities="CAPABILITY_IAM",
+            s3_prefix="integ_deploy",
+            s3_bucket=self.s3_bucket.name,
+            image_repository=self.ecr_repo_name,
+            force_upload=True,
+            notification_arns=self.sns_arn,
+            parameter_overrides="ShardCountParameter=1",
+            kms_key_id=self.kms_key,
+            no_execute_changeset=False,
+            tags="integ=true clarity=yes foo_bar=baz",
+            confirm_changeset=False,
+        )
+
+        deploy_process_execute = run_command(deploy_command_list)
+        self.assertEqual(deploy_process_execute.process.returncode, 1)
+
+        stderr = deploy_process_execute.stderr.strip()
+        self.assertIn(
+            bytes(
+                f"Error: Failed to create/update the stack: {stack_name}, Waiter StackCreateComplete failed: "
+                f'Waiter encountered a terminal failure state: For expression "Stacks[].StackStatus" '
+                f'we matched expected path: "ROLLBACK_COMPLETE" at least once',
+                encoding="utf-8",
+            ),
+            stderr,
+        )
+
+    @skip("skip until cloudformation bug is fixed")
```

Comment:
once cloudformation's bug is fixed, we can just remove this line and hopefully it should just work